src/main/java/com/bootcamp/demo/controller/CardController.java

```diff
@@ -0,0 +1,37 @@
+package com.bootcamp.demo.controller;
+
+import com.bootcamp.demo.model.Card;
+import com.bootcamp.demo.service.CardService;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.cloud.FirestoreClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/card")
+public class CardController {
+    private Firestore firestoreDB;
```

Comment:
Please check the comments from the "remove card" pull request: https://github.com/remus-lupin-team/payments-jb/pull/15
When the comments are addressed in the "base" branch please rebase your branch to get the latest commits.

Reply:
merged from parent branch "remove card" and adapted to its implementation (firebase dao).
feature tested and still works as expected.